lib/matplotlib/axes/_axes.py

```diff
@@ -3317,6 +3317,8 @@ def get_next_color():
                 else:
                     raise TypeError(
                         'autopct must be callable or a format string')
+                if textprops.get("usetex", mpl.rcParams["text.usetex"]):
```

Comment:
```suggestion
                if mpl._val_or_rc(textprops.get("usetex"), "text.usetex"):
```

Sorry to do this to you, but I remembered that we actually have a method to do these kinds of lookups now (it's pretty new).

The advantage of the implementation in the method as opposed to the `.get(..., rcParams[...])` way is that it will correctly handle the case where `None` is passed explicitly in the `textprops` dictionary. (i.e. resolving to the RC param, whereas the `.get(..., rcParams[...])` way will treat it as `False` because it retrieves the value `None` out of the dictionary and not use the default value.